core/src/utilities/helpers/generic-helpers.js

```diff
@@ -252,7 +252,14 @@ class GenericHelpersClass {
   }
 
   getContentAreaHeight /* istanbul ignore next */() {
-    return this.getInnerHeight() - LuigiElements.getShellbar().clientHeight;
+    const contentAreaHeight = this.getInnerHeight() - this.getShellbarHeight();
+    return contentAreaHeight;
+  }
+
+  getShellbarHeight() {
```

Comment:
To avoid future bugs, we usually add unit tests for each new function that is added/implemented. 

Could you please add a unit test for the new **getShellbarHeight** function that you added. 
Unit tests for this function should fit under `core/test/utilities/helpers/generic-helpers.spec.js`. You can check the file for similar examples.

Reply:
I added unit tests for the new `getShellbarHeight` function. See [5fc1aa8](https://github.com/SAP/luigi/pull/3301/commits/5fc1aa8050f5997c0d2064e58bb77ee2d18acb29).